pkg/logs/docker/docker.go

```diff
@@ -28,18 +30,76 @@ func ParseMessage(msg []byte) (string, string, []byte, error) {
 		return "", "", nil, errors.New("Can't parse docker message: expected a 8 bytes header")
 	}
 
+	preMessageLength := getPreMessageLength(msg)
+	if len(msg[preMessageLength:]) > maxDockerBuffer {
+		msg = removePartialHeaders(msg, preMessageLength)
```

Comment:
Wondering whether we should always call this method and add the message length condition there. WDYT ? The code would be more robust.

Reply:
Also, you would not need to pass `preMessageLength` as a parameter.